packages/remix-dev/vite/resolve-file-url.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from "node:path";
+
+export const resolveFileUrl = (
+  // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+  vite: typeof import("vite"),
```

Comment:
Note to self: If all callers of this util are async, can we make this util async and inline the dynamic import of Vite?